src/main/java/seedu/address/model/item/VisualizeList.java

```diff
@@ -23,7 +23,7 @@ public boolean add(Item item) {
         if (hasItem(item)) {
             return false;
         } else {
-            return list.add(item);
+            return super.add(item);
```

Comment:
Good work, although I'm curious about why there is an arraylist in this class itself? There are also some methods such as doAdd, getList, doSet, doRemove, get and size that relies on the list in this class, whereas other methods uses the list in the superclass. Is there a difference between those?

Reply:
This is the advised way to implement the ModifiableObservableList as mentioned here: https://openjfx.io/javadoc/11/javafx.base/javafx/collections/ModifiableObservableListBase.html

The actual use of doAdd, doRemove and doSet is also lost on me as I am not able to find a use for them at the moment.